CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Fixed
 * `SyncSession::pause()` could hold a reference to the database open after shutting down the sync session, preventing users from being able to delete the realm. ([#6372](https://github.com/realm/realm-core/issues/6372), since v13.3.0)
+* Fix a data race in `DB::VersionManager`. If one thread committed a write transaction which increased the number of live versions above the previous highest seen during the current session at the same time as another thread began a read, the reading thread could read from a no-longer-valid memory mapping ([PR #6411](https://github.com/realm/realm-core/pull/6411), since v13.0.0).
```

Comment:
How could end users experience this? @tgoyne could this be rephrased in end user terms that could easily be put in the SDK release notes?

Reply:
Good to know, then maybe we should add. "This hasn't been reported by any users" or similar?

Reply:
Good to know, then maybe we should add. "This hasn't been reported by any users" or similar?

Reply:
Good to know, then maybe we should add. "This hasn't been reported by any users" or similar?